tests/plugins/azure_storage_key_test.py

```diff
@@ -12,6 +12,10 @@ class TestAzureStorageKeyDetector:
                 'AccountKey=lJzRc1YdHaAA2KCNJJ1tkYwF/+mKK6Ygw0NGe170Xu592euJv2wYUtBlV8z+qnlcNQSnIYVTkLWntUO1F8j8rQ==',  # noqa: E501
                 True,
             ),
+            (
```

Comment:
did you add the false positives we got last time ? (to the tests?)

Reply:
added and fixed the test